fdbclient/FileBackupAgent.actor.cpp

```diff
@@ -5183,7 +5189,8 @@ class FileBackupAgentImpl {
 	}
 
 	ACTOR static Future<Optional<Version>> getLastRestorable(FileBackupAgent* backupAgent,
-	                                                         Reference<ReadYourWritesTransaction> tr, Key tagName,
+	                                                         Reference<ReadYourWritesTransaction> tr,
+	                                                         Key tagName,
```

Comment:
It's probably a clang-format version issue, so please revert this and the following formatting changes.

Reply:
Done